examples/rapids-image/app-image-config-input.json

```diff
@@ -0,0 +1,16 @@
+{
+    "AppImageConfigName": "custom-rapids-image-config",
+    "KernelGatewayImageConfig": {
+        "KernelSpecs": [
+            {
+                "Name": "rapids",
+                "DisplayName": "RAPIDS"
+            }
+        ],
+        "FileSystemConfig": {
+            "MountPath": "/home/",
```

Comment:
Is it intentional that this is `/home` and not the usual `/home/$USER`? The `$USER` will change depending on what user is the default in this image. In this image, the $USER is root so this is `/root/`

Reply:
Users with this UID and GID don't exist in this image. From what I can tell this should be 0,0

```
docker run -it rapidsai/rapidsai bash
A JupyterLab server has been started!
To access it, visit http://localhost:8888 on your host machine.
Ensure the following arguments were added to "docker run" to expose the JupyterLab server to your host machine:
      -p 8888:8888 -p 8787:8787 -p 8786:8786
Make local folders visible by bind mounting to /rapids/notebooks/host
(rapids) root@f5c87b9f6dc6:/rapids/notebooks# id -u
0
(rapids) root@f5c87b9f6dc6:/rapids/notebooks# id -g
0
```

Reply:
changed, thanks for checking the UID and GID, will update in the config file